docs/home/howto/OS_PATCHING.md

```diff
@@ -57,7 +57,7 @@ For more information, refer to [AWS Systems Manager User Guide](https://docs.aws
 For Epiphany >= v1.2 we recommend the following image (urn):
 
 - RHEL: `RedHat:RHEL:7-LVM:7.9.2021051701` (kernel 3.10.0-1160.el7.x86_64),
-- Ubuntu: `Canonical:UbuntuServer:18.04-LTS:18.04.202109130` (kernel 5.4.0-1058-azure).
+- Ubuntu: `Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:20.04.202112020` (kernel 5.11.0-1021-azure).
```

Comment:
Should also be updated then from @przemyslavic comment below.

Reply:
Yeah, I indicated only 2 places (as an example for AWS and Azure) but it needs to be changed in all places where it occurs.